libs/wingc/src/type_check/jsii_importer.rs

```diff
@@ -194,10 +203,7 @@ impl<'a> JsiiImporter<'a> {
 			return;
 		}
 
-		debug!(
-			"Type {} is unsupported or not found in the type system, skipping",
-			type_fqn
-		);
+		panic!("Type {} was not found in the type system, skipping", type_fqn);
```

Comment:
No longer skipping its blowing up 😁 
```suggestion
		panic!("Type {} was not found in the type system", type_fqn);
```